x-pack/test/functional/apps/lens/group6/annotations.ts

```diff
@@ -101,5 +110,98 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       await testSubjects.existOrFail('xyVisGroupedAnnotationIcon');
     });
+
+    describe('library annotation groups', () => {
```

Comment:
Can we get them to the flaky runner and run them for 100 times? Just to be sure that they are not flaky

Reply:
Definitely. I was hoping to add more before doing this, but given my local situation I don't know when that will happen. I'll go ahead with the tests I already wrote here.